Jenkinsfile

```diff
@@ -291,7 +289,8 @@ pipeline {
           expression { return env.ONLY_DOCS == "false" }
           anyOf {
             changeRequest()
-            expression { return !params.Run_As_Master_Branch }
+            expression { return params.integrationtests_ci }
+            expression { return env.GITHUB_COMMENT?.contains('integration tests') }
```

Comment:
Can you add this option in: https://github.com/elastic/apm-agent-java/pull/2082/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R31


```
issueCommentTrigger('(?i)(.*(?:jenkins\\W+)?run\\W+(?:the\\W+)?(?:(compatibility|benchmark|integration)\\W+)?tests(?:\\W+please)?.*|^/test(?:\\W+.*)?$)')
```